src/Features/Core/Portable/EncapsulateField/AbstractEncapsulateFieldService.cs

```diff
@@ -244,27 +245,35 @@ private async Task<Result> EncapsulateFieldAsync(IFieldSymbol field, Document do
             if (field.IsReadOnly)
             {
                 // Inside the constructor we want to rename references the field to the final field name.
-                var constructorSyntaxes = GetConstructorNodes(field.ContainingType).ToSet();
-                if (finalFieldName != field.Name && constructorSyntaxes.Count > 0)
+                var constructorLocations = GetConstructorLocations(field.ContainingType);
+                if (finalFieldName != field.Name && constructorLocations.Count > 0)
                 {
-                    solution = await Renamer.RenameSymbolAsync(solution,
-                        SymbolAndProjectId.Create(field, projectId),
-                        finalFieldName, solution.Options,
-                        location => constructorSyntaxes.Any(c => c.Span.IntersectsWith(location.SourceSpan)),
-                        cancellationToken: cancellationToken).ConfigureAwait(false);
+                    var initialLocations = await Renamer.GetRenameLocationsAsync(
+                        solution, SymbolAndProjectId.Create(field, projectId), solution.Options, cancellationToken).ConfigureAwait(false);
+
+                    var insideLocations = initialLocations.Filter(
+                        location => IntersectsWithAny(location, constructorLocations));
+
+                    solution = await Renamer.RenameAsync(
+                        insideLocations, finalFieldName, cancellationToken: cancellationToken).ConfigureAwait(false);
                     document = solution.GetDocument(document.Id);
 
                     var compilation = await document.Project.GetCompilationAsync(cancellationToken).ConfigureAwait(false);
 
                     field = field.GetSymbolKey().Resolve(compilation, cancellationToken: cancellationToken).Symbol as IFieldSymbol;
+                    constructorLocations = GetConstructorLocations(field.ContainingType);
```

Comment:
this was a bug i noticed.  after performing the first rename, we weren't recomputing the spans of the constructors in the new solution.